src/root.component.js

```diff
@@ -32,7 +32,7 @@ export default function Root(props) {
 
   useEffect(() => {
     if (
-      !localStorage.getItem("single-spa-playground") &&
+      !localStorage.getItem("single-spa-playground").application &&
```

Comment:
localStorage.getItem() returns a string, not an object. Which means that `str.application` will always be falsy? I think you need to JSON.parse() this before checking for the application property